### PR TITLE
Allow searching by more identifiers

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

File: estuary/models/koji.py [hunk 1]
```diff
@@ -28,6 +31,31 @@ class KojiBuild(EstuaryStructuredNode):
...
+        uid = identifier
+        if re.match(r'^\d+$', uid):
+            # The identifier is an ID
+            return cls.nodes.get_or_none(id_=uid)
+        elif uid.endswith('.src.rpm'):
+            # The identifer is likely an NVR with .src.rpm at the end, so strip that part of it
+            # so it can be treated like a normal NVR
+            uid = uid[:-8]
+
+        if len(uid.rsplit('-', 2)) == 3:
```
Comment (yashvardhannanavati) [inline estuary/models/koji.py]: Can we support this with another heuristic as well? This doesn't look bullet proof

File: tests/api/test_story.py [hunk 2]
```diff
@@ -491,24 +491,24 @@
...
+                'security_impact': 'None',
+                'security_sla': None,
+                'state': 'SHIPPED_LIVE',
+                'states': [
 
                 ],
-                'status_time':'2017-08-01T15:43:51+00:00',
-                'synopsis':'cifs-utils bug fix update',
-                'triggered_freshmaker_event':[
+                'status_time': '2017-08-01T15:43:51+00:00',
```
Comment (yashvardhannanavati) [inline tests/api/test_story.py]: @mprahl This is the reason I use sublime :P

File: estuary/models/koji.py (hunk 1 above)
Comment (mprahl) [reply estuary/models/koji.py]: I thought maybe there was, but I took the applicable code from:
https://github.com/release-engineering/kobo/blob/master/kobo/rpmlib.py#L166

Comment (yashvardhannanavati) [reply estuary/models/koji.py]: cool, +1

**Pull request merged**